discovery_engine_core/web-api/openapi.yaml

```
@@ -324,11 +324,15 @@ paths:
 components:
   schemas:
     DocumentId:
-      description: A document id can be any non-empty UTF-8 string that does not contain the null byte.
+      description: |-
+        A document id can be any non-empty UTF-8 string that does not contain the null byte.
+        It may consist of digits, Latin letters, underscores, colons, minus signs, at signs, and dots.
       type: string
-      example: "document id"
+      example: "document_id"
     DocumentPropertyId:
-      description: A document property id can be any non-empty UTF-8 string that does not contain the null byte.
+      description: |-
+        A document property id can be any non-empty UTF-8 string that does not contain the null byte.
+        It may consist of digits, Latin letters, underscores, colons, minus signs, at signs, and dots.
       type: string
       example: "document property id"
```

🔧 space is not valid anymore
```suggestion
      example: "document_property_id"
```